doc/contributing/security-release-process.md

```diff
@@ -19,14 +19,14 @@ steps listed in the process as outlined in
 The current security stewards are documented in the main Node.js
 [README.md](https://github.com/nodejs/node#security-release-stewards).
 
-| Company    | Person   | Release Date |
-| ---------- | -------- | ------------ |
-| NearForm   | Matteo   | 2021-Oct-12  |
-| Datadog    | Bryan    | 2022-Jan-10  |
-| RH and IBM | Joe      | 2022-Mar-18  |
-| NearForm   | Matteo   |              |
-| Datadog    | Vladimir |              |
-| RH and IBM | Michael  |              |
+| Company    | Person           | Release Date |
+| ---------- | ---------------- | ------------ |
+| NearForm   | Matteo           | 2021-Oct-12  |
+| Datadog    | Bryan            | 2022-Jan-10  |
+| RH and IBM | Joe              | 2022-Mar-18  |
+| NearForm   | Matteo / Rafael  |              |
+| Datadog    | Vladimir         |              |
+| RH and IBM | Michael          |              |
```

Comment:
```suggestion
| Company    | Person          | Release Date |
| ---------- | --------------- | ------------ |
| NearForm   | Matteo          | 2021-Oct-12  |
| Datadog    | Bryan           | 2022-Jan-10  |
| RH and IBM | Joe             | 2022-Mar-18  |
| NearForm   | Matteo / Rafael |              |
| Datadog    | Vladimir        |              |
| RH and IBM | Michael         |              |
```
I think this will make the linter happy.